src/actions/guides/database/querying.cr

```diff
@@ -355,7 +374,7 @@ class Guides::Database::Querying < GuideAction
     You can use this to help refine your association.
 
     ```crystal
-    UserQuery.new.join_taks.tasks { |task_query|
+    UserQuery.new.join_tasks.where_tasks { |task_query|
```

Comment:
`join_tasks` is no  longer needed. May be worth mentioning that an inner join is added automatically

Reply:
Maybe  this should link to the sections further down with more examples? Also the sentence on 371 snow out of date.It no longer passes a block

Reply:
oh good catch. It doesn't take a block either!